webapp/src/index.tsx

```diff
@@ -281,6 +282,10 @@ export default class Plugin {
             return slashCommandsHandler(store, joinCall, message, args);
         });
 
+        registry.registerDesktopNotificationHook?.(async (post, msgProps, channel, teamId, args) => {
+            return desktopNotificationHandler(store, post, channel, args);
+        });
```

Comment:
I assume it's not a problem but just confirming that this won't have any effect on unsupported MM versions.

Reply:
👍 The `?.` prevents a crash, and nothing will happen if this is run on an unsupported version.